### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by tw-circleci**

### CODE REVIEW

**Review by dozortsev — APPROVED** [review]

File: packages/components/i18n/it.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "neptune.CloseButton.ariaLabel": "Close",
-  "neptune.Pagination.ariaLabel": "Pagination navigation",
-  "neptune.PaginationLink.ariaLabel.active": "Go to page {pageNumber}",
-  "neptune.PaginationLink.ariaLabel.inactive": "Current page, page {pageNumber}"
-}
+  "neptune.CloseButton.ariaLabel": "Chiudi",
+  "neptune.Pagination.ariaLabel": "Navigazione",
+  "neptune.PaginationLink.ariaLabel.active": "Via a pagina {pageNumber}",
```
Comment (andreapiras) [inline packages/components/i18n/it.json:4]: This is not correct

Comment (dozortsev) [reply packages/components/i18n/it.json:4]: @andreapiras could you please suggest fix based on English message?

Comment (andreapiras) [reply packages/components/i18n/it.json:4]: Vai a pagina {pageNumber}

Comment (dozortsev) [inline packages/components/i18n/it.json:4]:
```suggestion
  "neptune.PaginationLink.ariaLabel.active": "Vai a pagina {pageNumber}",
```

**Pull request merged**